kubeshop/tracetest
# feat: adding instana as a tracing backend

This PR adds Instana as an OTLP-based tracing backend for Tracetest

## Changes

- Added FE icons and code for Instana on Wizard and `Settings > Tracing Backend` page
- Added Instana as a Datastore on Backend API
- Added example using Instana as a tracing backend
- Added documentation page under `./docs/docs/configuration/connecting-to-data-stores`

## How to manually test it

1. Check out the `add/instana-integration-v1`
2. Create a new docker image for this code with `make build-docker`
3. On terminal to `./examples/quick-start-instana-nodejs` and create an `.env` file with the following data:
```
 

```
4. Run the command `TAG=dev docker compose up -d`
5. After everything is set, you can run a test using `tracetest run test -f ./test-api.yaml --server-url=http://localhost:11633`
6. You see the UI changes by accessing `http://localhost:11633` on your browser.


## Checklist


## Loom video

https://www.loom.com/share/421fd9bc85174d4db89ac1c9a50635c4?sid=1344e71e-b8db-42ac-98ec-39a1b5e0d736
